controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -184,6 +185,18 @@ func buildArgs(desired *flowslatest.FlowCollectorSpec) []string {
 			args = append(args, "--loki-ca-path", helper.GetCACertPath(&desired.Loki.TLS, lokiCerts))
 		}
 	}
+
+	statusTLS := helper.LokiStatusTLS(&desired.Loki)
+	if statusTLS.Enable {
+		if statusTLS.InsecureSkipVerify {
+			args = append(args, "-loki-status-skip-tls")
```

Comment:
is this a typo it should be `--loki-status-skip-tls` ?

Reply:
I think we should harmonize everything with a single minus character as it's already the case for most of these args. I will also fix the existing `--loki-ca-path`

Thanks for pointing that !